packages/create-gatsby/src/index.ts

```diff
@@ -167,6 +174,23 @@ ${center(c.blueBright.bold.underline(`Welcome to Gatsby!`))}
 
   const data = await enquirer.prompt(questions)
 
+  trackCli(`CREATE_GATSBY_SELECT_OPTION`, {
+    name: `project_name`,
+    valueString: sha256(data.project),
```

Comment:
I assume this is to give us a sort of pseudo-id to store with the data below? (So we could see what styling they might choose with a certain CMS for example)

Otherwise I'd ask why we need to hash the name and store it.

Reply:
yea, we can still categorize sites even though we don't know what it's called and won't accidentally carry any PII